ext/rotoscope/rotoscope.c

```diff
@@ -13,7 +13,9 @@
 #include "strmemo.h"
 #include "tracepoint.h"
 
-VALUE cRotoscope, cTracePoint;
+VALUE cRotoscope, cTracePoint, cInstructionSeq;
+ID id_of, id_label;
+unsigned char log_buffer[500];
```

Comment:
Is 500 characters arbitrary? Or do we have a guarantee that the log buffer will never exceed 500 chars?

Reply:
Yeah it's arbitrary. I chose it by looking at the longest line in our Rotoscope output and adding some extra space. Though, after seeing new code like `Pricing::Core::Appliers::Embedded::Applier::LineItemApplier` I'm wondering if I should bump this a bit higher.